packages/perseus-editor/src/components/graph-settings.tsx

```diff
@@ -502,7 +502,13 @@ const GraphSettings = createReactClass({
                                     {value: "grid", content: "Grid"},
                                     {value: "none", content: "None"},
                                 ]}
-                                onChange={this.change("markings")}
+                                onChange={(value) =>
+                                    Changeable.changeSingleProp(
+                                        this,
+                                        "markings",
```

Comment:
This causes a type error because `GraphSettings` is still using `createReactClass`. 😞